other/field-types/array.md

```diff
@@ -2,15 +2,9 @@
 
 An `array` field has its own schema, and allows the user to create one or more objects that have the fields in that schema. These objects are stored as an array.
 
-This is useful for collections that clearly belong to a parent object, such as multiple homes for a person or tabs in a widget, and do not have any other relationships to other objects. If the objects in the array are also related to other types of objects, you should be using joins instead.
+This is useful for collections that clearly belong to a parent object, such as multiple homes for a person or tabs in a widget, and do not have any other relationships to other objects.
```

Comment:
I don't love the homes example. How about "email addresses for a business."

Reply:
What, isn't owning multiple homes a totally normal thing that every person does? ;)